### PR TITLE
feat: added OpenSearch Vector Database

### PR DESCRIPTION
**Pull request by curlyfro**
fix: Titan embedding model response
fix: other minor fixes

### CODE REVIEW

File: src/Databases/OpenSearch/src/LangChain.Databases.OpenSearch.csproj [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    <PropertyGroup>
+        <TargetFrameworks>net4.6.2;netstandard2.0;net6.0;net7.0;net8.0</TargetFrameworks>
+        <SignAssembly>false</SignAssembly>
+        <NoWarn>$(NoWarn);CA1720;CA1510;CA2100</NoWarn>
+    </PropertyGroup>
+
+    <PropertyGroup Label="NuGet">
+        <Description>Postgres for LangChain.</Description>
+        <PackageTags>$(PackageTags);postgres</PackageTags>
```
Comment (HavenDV) [inline src/Databases/OpenSearch/src/LangChain.Databases.OpenSearch.csproj:11]: replace postgres to opensearch

Comment (curlyfro) [reply src/Databases/OpenSearch/src/LangChain.Databases.OpenSearch.csproj:11]: i just saw that

Comment (HavenDV) [reply src/Databases/OpenSearch/src/LangChain.Databases.OpenSearch.csproj:11]: Also for PackageTags

**Comment by HavenDV** [comment]
image

Please fix errors. I can fix warnings after merge.

**Pull request merged**